pkg/storage/types/pvc.go

```diff
@@ -195,3 +198,36 @@ func VolumeReadyToAttachToNode(namespace string, volume virtv1.Volume, dataVolum
 	}
 	return ready, wffc, nil
 }
+
+func GeneratePVC(size *resource.Quantity, claimName, namespace, storageClass, accessMode string, blockVolume bool) *k8sv1.PersistentVolumeClaim {
```

Comment:
I think the word used in kubevirt for creating the `spec` is `render` not `generate`.

Reply:
as in RenderPod, etc...